src/utils.jl

```diff
@@ -466,11 +469,28 @@ function cross_section_points(P::GeoData; Depth_level = nothing, Lat_level = not
             error("Also define End coordinates if you indicate starting lon/lat value")
         end
 
-        # choose projection point based on Start and End coordinates of the profile
-        p_Point = ProjectionPoint(Lat = 0.5 * (Start[2] + End[2]), Lon = 0.5 * (Start[1] + End[1]))
+        p_Point = ProjectionPoint(Lat = 0.5 * (Start[2] + End[2]), Lon = 0.5 * (Start[1] + End[1])) # choose the projection point as the midpoint of the profile
 
         # convert P to UTM Data
-        P_UTM = convert2UTMzone(P, p_Point) # convert to UTM
+        # to avoid projection issues, reduce the given point data set to a set that is located abound the desired profile
+        # to choose the subset, we use a box around the profile with the profile width being added to the start and end points of the profile
+        # approximate formula: 
+        # Latitude: 1 deg = 110.574 km --> 1 km = 1/110.574 deg
+        # Longitude: 1 deg = 111.320*cos(latitude) km --> 1km = 1/ 111.320 / cos(latitude) 
+        lat_add = 1.1 * ustrip(uconvert(u"km", section_width))/110.574 # multiply with 1.1 to make sure the box is large enough
+        lat_start = minimum([Start[2],End[2]]) - lat_add;
+        lat_end   = maximum([Start[2],End[2]])   + lat_add;
+        
+        lon_add = 1.1*ustrip(uconvert(u"km", section_width))/111.3209
+
+        lon_start = minimum([Start[1],End[1]]) - lon_add/cos(deg2rad(minimum([Start[1],End[1]])))
+        lon_end   = maximum([Start[1],End[1]]) + lon_add/cos(deg2rad(maximum([Start[1],End[1]])))
+
+        ind = findall( (lon_start .< P.lon.val .< lon_end) .& (lat_start .< P.lat.val .< lat_end))
```

Comment:
```suggestion
        ind = @. (lon_start < P.lon.val < lon_end) & (lat_start < P.lat.val < lat_end)
```
I think you can get away without `findall` here